zcash_client_sqlite/src/wallet/init/migrations/addresses_table.rs

```diff
@@ -52,6 +52,8 @@ impl<P: consensus::Parameters> RusqliteMigration for Migration<P> {
                 account INTEGER NOT NULL,
                 diversifier_index_be BLOB NOT NULL,
                 address TEXT NOT NULL,
+                sapling_receiver TEXT,
```

Comment:
We can't store Orchard receivers as `TEXT`, so we shouldn't store Sapling this way either. But as this column is unused, we should just remove it:
```suggestion
```